lib/Models/Cesium.js

```diff
@@ -757,6 +757,35 @@ Cesium.prototype._buildPickedFeatures = function(providerCoords, pickPosition, e
     return result;
 };
 
+Cesium.prototype.getBoundingBoxCoords = function() {
```

Comment:
Doesn't this do the same thing as `getCurrentExtent`, except this one won't work if the globe isn't in the corners of the screen? :)